Configure

```diff
@@ -997,8 +1000,8 @@ if ($target eq "HASH") {
     exit 0;
 }
 
-print "Configuring OpenSSL version $config{version} ($config{version_num}) ";
-print "for $target\n";
+print "Configuring OpenSSL version '$config{full_version}' ";
+print "for target '$target'\n";
```

Comment:
I see no need for the added quotes

Reply:
I thought it might make version and target more visible in the output, see sample output above. But if you don't like it, I'll remove it again.

Reply:
I'd rather you dropped them.

Reply:
Ok, I removed them (f02a102082c16d3281de68e14fbccfb67ec501ed).